lib/parseConfig/index.js

```diff
@@ -8,6 +8,15 @@ var getModel 			= require('./getModel');
 
 module.exports = function (config) {
 
+	if (_.isString(config)) {
+		try {
+			config = JSON.parse(config);
+		} catch (e) {
+			throw new Error('Failed to parse connector test data.');
```

Comment:
Because it will only ever be passed as a string when being tested via the UI (that's when it's sent as a string)